components/message_with_additional_content.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import {Posts} from 'mattermost-redux/constants';
+
+import PostBodyAdditionalContent from 'components/post_view/post_body_additional_content';
+import PostMessageContainer from 'components/post_view/post_message_view';
+
+export default function MessageWithAdditionalContent({post, previewCollapsed, previewEnabled, isEmbedVisible}) {
+    let msg;
+    const messageWrapper = (
+        <PostMessageContainer
+            post={post}
+            isRHS={true}
+            hasMention={true}
+        />
+    );
+    if (post.state === Posts.POST_DELETED) {
```

Comment:
Was just checking post_body component. Does plugin support extend to RHS as well?

https://github.com/mattermost/mattermost-webapp/blob/e7bcf85aae63017a0f537f32011fa67387d1c6cc/components/post_view/post_body/post_body.jsx#L256

Not a block for the PR as I am not sure what is the behaviour here is

Reply:
@sudheerDev Good point. From [the comment in where it was added](https://github.com/mattermost/mattermost-webapp/pull/306#discussion_r151159108) it sounds like it could be added here too. @jwilander Can you confirm that the same plugin check should be added?

Reply:
Yeah it should be included here as well 👍

Reply:
Added.